api/envoy/config/rbac/v2/rbac.proto

```diff
@@ -201,17 +201,21 @@ message Principal {
     // Authenticated attributes that identify the downstream.
     Authenticated authenticated = 4;
 
-    // A CIDR block that describes the downstream direct remote address.
+    // A CIDR block that describes the downstream IP.
+    api.v2.core.CidrRange source_ip = 5 [deprecated = true];
+
+    // A CIDR block that describes the downstream remote/origin address.
     // Note: This is always the physical peer even if the
-    // :ref:`remote_ip <envoy_api_field_config.rbac.v2.Principal.remote_ip>` is inferred from for example
-    // the x-forwarder-for header, proxy protocol, etc.
-    api.v2.core.CidrRange source_ip = 5;
+    // :ref:`remote_ip <envoy_api_field_config.rbac.v2.Principal.remote_ip>` is inferred
+    // from for example the x-forwarder-for header, proxy protocol, etc.
+    api.v2.core.CidrRange peer_ip = 10;
```

Comment:
Should this be renamed `direct_remote_ip` or similar, to be consistent with other uses of this concept?